src/main/scala/akka/stream/alpakka/xlsx/SstStreamer.scala

```diff
@@ -55,30 +55,30 @@ object SstStreamer {
         var count                              = 0
         var si                                 = false
```

Comment:
Is that in Spanish?

Reply:
No, `si` stands for "string item". More: https://docs.microsoft.com/en-us/office/open-xml/working-with-the-shared-string-table.